boards/posix/native_posix/irq_ctrl.h

```diff
@@ -26,7 +26,7 @@ uint8_t hw_irq_ctrl_get_prio(unsigned int irq);
 int hw_irq_ctrl_get_highest_prio_irq(void);
 uint32_t hw_irq_ctrl_get_current_lock(void);
 uint32_t hw_irq_ctrl_change_lock(uint32_t new_lock);
-uint64_t hw_irq_ctrl_get_irq_status(void);
+u64_t hw_irq_ctrl_get_irq_status(void);
```

Comment:
wouldn't a zephyr/types.h include be missing for this? (to keep this header self-contained)